en/orm/retrieving-data-and-resultsets.rst

```diff
@@ -497,8 +497,9 @@ load fields off of contained associations, you can use ``autoFields()``::
 
     // Select id & title from articles, but all fields off of Users.
     $query->select(['id', 'title'])
-        ->contain(['Users'])
-        ->autoFields(true);
+        ->contain(['Users' => function($q) {
+			return $q->autoFields(true);
+		          }]);
```

Comment:
I only made a test on belongsTo association.... but the original proposal was not working according to the comment on line 498, it generates fields for both association AND main entity, canceling the select() statement.